applications/FluidDynamicsApplication/tests/TwoFluidStaticPoolTest/TwoFluidStaticPoolTest3D.mdpa

```diff
@@ -0,0 +1,19460 @@
+ 1198   0.1363636364   0.2272727273   0.9090909091
+ 1199   0.4090909091  -0.1363636364   0.2727272727
+ 1200   0.2272727273   0.1363636364   0.8181818182
+ 1201  -0.0454545455  -0.4090909091   0.5454545455
+ 1202   0.0454545455  -0.4090909091   0.4545454545
+ 1203   0.0454545455  -0.5000000000   0.1818181818
+ 1204  -0.3181818182  -0.5000000000   0.5454545455
+ 1205   0.5000000000  -0.0454545455   0.1818181818
+ 1206   0.5000000000   0.3181818182   0.5454545455
+ 1207  -0.1363636364  -0.2272727273   0.8181818182
+ 1208  -0.0454545455  -0.0454545455   0.9090909091
+ 1209   0.0454545455   0.0454545455   0.9090909091
+ 1210   0.3181818182   0.1363636364   0.7272727273
+ 1211   0.4090909091  -0.0454545455   0.4545454545
+ 1212  -0.3181818182  -0.0454545455   1.0000000000
+ 1213   0.0454545455   0.3181818182   1.0000000000
+ 1214  -0.1363636364  -0.3181818182   0.7272727273
+ 1215   0.2272727273  -0.3181818182   0.3636363636
+ 1216   0.3181818182  -0.2272727273   0.3636363636
+ 1217   0.4090909091   0.0454545455   0.5454545455
+ 1218  -0.1363636364   0.0454545455   1.0000000000
+ 1219  -0.0454545455   0.1363636364   1.0000000000
+ 1220  -0.0454545455  -0.5000000000   0.3636363636
+ 1221  -0.1363636364  -0.5000000000   0.4545454545
+ 1222   0.5000000000   0.0454545455   0.3636363636
+ 1223   0.5000000000   0.1363636364   0.4545454545
+ 1224   0.1363636364  -0.1363636364   0.7272727273
+ 1225   0.2272727273  -0.1363636364   0.6363636364
+ 1226   0.3181818182  -0.3181818182   0.0000000000
+ 1227  -0.5000000000  -0.3181818182   0.8181818182
+ 1228   0.3181818182   0.5000000000   0.8181818182
+ 1229   0.1363636364  -0.2272727273   0.6363636364
+ 1230  -0.4090909091  -0.3181818182   0.8181818182
+ 1231   0.3181818182  -0.3181818182   0.0909090909
+ 1232   0.3181818182   0.4090909091   0.8181818182
+ 1233   0.0454545455  -0.2272727273   0.7272727273
+ 1234   0.2272727273  -0.4090909091   0.0000000000
+ 1235  -0.5000000000  -0.2272727273   0.9090909091
+ 1236   0.4090909091   0.5000000000   0.7272727273
+ 1237   0.2272727273  -0.0454545455   0.7272727273
+ 1238   0.4090909091  -0.2272727273   0.0000000000
+ 1239   0.2272727273  -0.2272727273   0.5454545455
+ 1240  -0.5000000000  -0.4090909091   0.7272727273
+ 1241   0.2272727273   0.5000000000   0.9090909091
+ 1242  -0.1363636364  -0.1363636364   0.9090909091
+ 1243   0.4090909091  -0.1363636364   0.3636363636
+ 1244  -0.4090909091  -0.2272727273   0.9090909091
+ 1245   0.1363636364  -0.4090909091   0.3636363636
+ 1246   0.2272727273  -0.4090909091   0.0909090909
+ 1247   0.4090909091   0.1363636364   0.6363636364
+ 1248   0.4090909091   0.4090909091   0.7272727273
+ 1249  -0.4090909091  -0.4090909091   0.7272727273
+ 1250  -0.1363636364  -0.4090909091   0.6363636364
+ 1251   0.1363636364   0.1363636364   0.9090909091
+ 1252   0.2272727273   0.4090909091   0.9090909091
+ 1253   0.4090909091  -0.2272727273   0.0909090909
+ 1254  -0.2272727273  -0.0454545455   1.0000000000
+ 1255   0.0454545
```

Comment:
this is a huuuuge test
Does it need to be this big?

Reply:
A considerably smaller test was set up